module/js/Api.js

```diff
@@ -0,0 +1,12 @@
+import {SharedConsts} from "../shared/SharedConsts.js";
```

Comment:
n.b.: from here until `SharedConsts.js` is just diff from copy-pasting code between files